lib/helpers.ts

```diff
@@ -1,5 +1,6 @@
 import { TransactionType } from "@/store/transactionsSlice";
 import { WalletType } from "./types";
+import { appConfig } from "./config";
```

Comment:
@MayankMittal1 seems like it's a circular dependency issue in the code, which is causing Vercel deployment to fail. Please try to move `getTokenOnFuse` function in a new file, such as `helper-bridge.ts`.